### PR TITLE
i think this is more correct, plus admission dates are mostly off

### PR DESCRIPTION
**Pull request by mgoldhirsh**
## Describe your changes
Please include a summary of any changes.
noticed a lot of proc dates 1900-01-01. turn out there was a coalse that prioritizes admission date first. also this didnt use line date, which seems the most germain to me, and for things like HEDIS, exact date matters, so not sure we want this to be how we calc proc dates.

I brought this up to Tuva Health, and will put in a PR if they agree

## How has this been tested?
Please describe the tests you ran to verify your changes.  Provide instructions or code to reproduce output.

## Reviewer focus
Please summarize the specific items you’d like the reviewer(s) to look into.

## Checklist before requesting a review
- [ ] I have added at least one Github label to this PR (bug, enhancement, breaking change,...)
- [ ] My code follows [style guidelines](https://thetuvaproject.com/guides/contributing/style-guide)
- [ ] (New models) [YAML files](https://github.com/tuva-health/tuva/blob/main/models/hcc_suspecting/hcc_suspecting_models.yml) are categorized by sub folder and models listed in alphabetical order
- [ ] (New models) I have added a [config](https://github.com/tuva-health/tuva/blob/main/models/hcc_suspecting/final/hcc_suspecting__list.sql) to each new model to enable it for claims and/or clinical data
- [ ] (New models) I have added the variable `tuva_last_run` to the final output
- [ ] (Optional) I have recorded a Loom to explain this PR

### Package release checklist
- [ ] I have updated [dbt docs](https://www.notion.so/tuvahealth/Building-dbt-Docs-16df2f00df244f29b9d6756d8adbc2d9)
- [ ] I have updated the version number in the `dbt_project.yml`

### CODE REVIEW

File: models/core/staging/core__stg_claims_procedure.sql
Comment (justinmills) [inline models/core/staging/core__stg_claims_procedure.sql:18]: Have we checked with the tuva community (or newer versions of this model) to see if this is correct? Also, wondering if we should just add this as the first item in the coalesce, or if it's better to have no date given the other dates being coalesced may produce incorrect results?

**Review by justinmills — APPROVED** [review]
Approving to unblock assuming my q/suggestion is not concerning.

File: models/core/staging/core__stg_claims_procedure.sql
Comment (mgoldhirsh) [reply models/core/staging/core__stg_claims_procedure.sql:18]: yeah, that was my first instinct, but I am more worried about correctness than missingness. And yes, it is still like this in Tuva's main branch. I reached out to Thu at Tuva for her thoughts on this

Comment (justinmills) [reply models/core/staging/core__stg_claims_procedure.sql:18]: Cool, yeah, probably best to skew towards correctness - if we do find the maintainers are reluctant to merge this in as-is, we can always set up another input var to allow for our custom behavior - that'd probably be more amenable to the community to be able to opt into this logic vs. have it be the default/only option. But let's see what they think.

**Pull request merged**
